problems/numberOfDigits.js

```diff
@@ -7,8 +7,17 @@
 
 */
 
-function numberOfDigits() {
-
+function numberOfDigits(n) {
+    let count = 0 
+    let str = n.toString()
```

Comment:
Is there a method we could use here that would give us the number of characters in a string?